src/coach/package.xml

```diff
@@ -37,7 +37,6 @@
   <!--   <build_depend>roscpp</build_depend> -->
```

Comment:
Remove boilerplate comments

Reply:
Done for all package.xml and CMakeLists.txt files